stories/2-Card.stories.js

```diff
@@ -49,6 +49,20 @@ export const ContextCard = () => (
   </ContextStyle>
 );
 
+export const DropdownMenu = () => (
+  <DropdownMenuCard>
+    <li>
+      <SubtleLink className="menu-link">OpenSaucedUser</SubtleLink>
+    </li>
+    <li>
+      <SubtleLink className="menu-link">Logout</SubtleLink>
+    </li>
+    <li>
+      <span className="app-info">v0.01</span>
+    </li>
+  </DropdownMenuCard>
+);
+
```

Comment:
Thanks so much add this to the storybook 🙏🏾

Reply:
Thanks! 😃